docs/examples/metrics/reader/README.rst

```diff
@@ -0,0 +1,34 @@
+View common scenarios
+=====================
+
+These examples show how to customize the metrics that are output by the SDK using configuration on metric readers. There are multiple examples:
+
+* preferred_aggregation: Shows how to configure the preferred aggregation for metric instrument types.
+* preferred_temporality: Shows how to configure the preferred temporality for metric instrument types.
```

Comment:
```suggestion
* `preferred_temporality.py`: Shows how to configure the preferred temporality for metric instrument types.
```

Reply:
I find it easier to understand if we specifically mention the file names, since these are also parameters for the `MetricReader` class.